### DISCUSSION TITLE
Actions Menu Uncaught Type Error in v18.0.0

### DISCUSSION THREAD

**Discussion by MickMorley** [jcorporation/myMPD#1355] [Q&A]
In the latest version, the Actions button causes an error `Uncaught TypeError: Cannot read properties of undefined (reading 'substr')` in the UI.  This occurs under the `Search` menu and `Browse` menu.  It only happens on individual songs, if you select the Actions button on an Album, it works.  The side pane displays.

I believe this issue started last week which is when 18.0.0 was released.  There was a container update today that I tried as well, but same issue.

Edit: Just tried a new installation (with a fresh work and cache directory) and the same issue persists.

![image](https://github.com/user-attachments/assets/2ff25493-704e-482f-9814-156379187ec1)

**Comment by jcorporation** [comment]
It is a bug. I opened an issue for it.
